source/templates.tex

```diff
@@ -5331,29 +5335,39 @@
 
 \pnum
 For a \grammarterm{noexcept-specifier} of a function template
-specialization or specialization of a member function of a class template, if
-the \grammarterm{noexcept-specifier} is implicitly instantiated because
-it is needed by another template specialization and the context that requires
-it depends on a template parameter, the point of instantiation of the
+specialization or specialization of a member function of a class template,
+the point of instantiation of the
 \grammarterm{noexcept-specifier} is the point of instantiation of the
-specialization that requires it. Otherwise, the point of instantiation for such
+specialization that requires it if
+\begin{itemize}
+\item the \grammarterm{noexcept-specifier} is implicitly instantiated because
+it is needed by another template specialization, and
```

Comment:
Can we get semantic linebreaks for the bullets, please?  ("because" in the next line)
See http://sembr.org